validator/entrypoint.sh

```diff
@@ -5,6 +5,22 @@ NETWORK="mainnet"
 VALIDATOR_PORT=3500
 WEB3SIGNER_API="http://web3signer.web3signer.dappnode:9000"
 
+
+if [[ "$EXIT_VALIDATOR" == "I want to exit my validators" ]]; then
```

Comment:
what would happen once the user has set this value to the env? it will attempt to exit the validators even if there are new uploads, which is really dangerous.

Reply:
We should be safed since we are defining the validators to exit in the variable `KEYSTORES_VOLUNTARY_EXIT`

Reply:
Not at all, the user MUST define the public keys in the KEYSTORES_VOLUNTARY_EXIT , but I think it would be a good practice to check that must have content on the KEYSTORES_VOLUNTARY_EXIT.